### PR TITLE
Feat: Improve dropdown fiat selector position, correct alignment

### LINKED ISSUE #279: [Task]: Improve mobile navbar and fiat selector alignment
### Task description

It would be a good idea to improve alignment in mobile in order to see correctly navbar and fiat selector.
image

We can also add a rotation to the chevron icon when Fiat selector is clicked like network switcher or utilities.
### Requirements

TBD

### Acceptance criteria

TBD

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [ ] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by rubalv96**
# Description of change
This PR aims to close #279

## Type of change
- Enhancement (a non-breaking change which adds functionality)

## Change checklist
- [X] My code follows the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: client/src/app/components/FiatSelector.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@
     color: var(--body-color);
     appearance: none;
 
+    @include phone-down {
+      padding: 0 24px 0 8px;
```
Comment (evavirseda) [inline client/src/app/components/FiatSelector.scss:30]: I think a small decrease in `fiat-selector_button` height, like 35px, would harmonize mobile navbars

**Review by evavirseda — CHANGES_REQUESTED** [review]
One small suggestion, the rest looks great to me :blush:

File: client/src/app/components/FiatSelector.scss (hunk 1 above)
Comment (rubalv96) [reply client/src/app/components/FiatSelector.scss]: thanks you, it looks great! 👍🏼

**Review by begonaalvarezd — APPROVED** [review]
LGTM 🎸

**Pull request merged**